src/setup/devnet/templates/remote/bor/bor-start.sh.njk

```diff
@@ -16,7 +16,7 @@ bor server \
   --port 30303 \
   --bor.heimdall "http://localhost:1317" \
   --http --http.addr '0.0.0.0' \
-  --ws --ws.port 8546 \
+  --ws --ws.addr '0.0.0.0' --ws.port 8546 --ws.api 'eth,txpool,net,web3,bor' --ws.vhosts='*' \
```

Comment:
Since this is merged: https://github.com/maticnetwork/bor/pull/574, `--ws.vhosts` could be changed to `--ws.origins`

Reply:
Sure, thank you!